fix(core): deprecate `moduleId` `@Component` property (#49496)
angular/angular: packages/core/schematics/migrations/remove-module-id/README.md

## RemoveModuleId migration

> As of Angular version 9, the `moduleId` property did no longer have any effect. This migration
> removes the field from all `@Directive` or `@Component` decorators.

Reviewer: ```suggestion
As of Angular version 9, the `moduleId` property has no effect. This migration
removes the field from all `@Directive` or `@Component` decorators.
```
